subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -982,7 +982,9 @@ static void handle_sensor_status(struct bt_mesh_model *mod,
 		 * able to turn on the light:
 		 */
 		if (srv->state == LIGHT_CTRL_STATE_STANDBY &&
-		    !atomic_test_bit(&srv->flags, FLAG_OCC_MODE)) {
+		    !atomic_test_bit(&srv->flags, FLAG_OCC_MODE) &&
```

Comment:
It seems like this code is for implementing condition in table 6.2.5.6 and not 6.2.5.11. Earlier code looks more correct. Perhaps you need additional `if` for STANDBY_AUTO

Reply:
2 new lines represent Standby Auto state. When the state machine transitions from Prolong to Standby, the state (`srv->state`) is set to `LIGHT_CTRL_STATE_STANDBY` and `FLAG_TRANSITION` is set. If the manual transition happens, `FLAG_MANUAL` is also set.